### ISSUE TITLE
Support Eureka

### ISSUE THREAD

**Issue by chaganiu** [bojand/ghz#241]
**Is your feature request related to a problem? Please describe.**
It would be great if we could get Eureka support for ghz so we can see how our services perform in real world situations through a load balancer.  I'm hoping this can help with the feature request: https://github.com/vadimi/grpc-client-cli/blob/master/internal/resolver/eureka/eureka_resolver.go

**Describe the solution you'd like**
Support for Eureka 

**Describe alternatives you've considered**
Right now we are hitting individual nodes directly.

**Additional context**
https://github.com/vadimi/grpc-client-cli/blob/master/internal/resolver/eureka/eureka_resolver.go

**Comment by bojand** [comment]
Yup support for load balancing has been requested in other issues, see #103. I agree it would be a useful feature. I will try and get to it eventually, but my time is limited at the moment.
